src/sentry/api/endpoints/organization_events_trends_v2.py

```diff
@@ -426,14 +392,6 @@ def parse_and_send_regressions_to_plaform(found_trending_events):
                 trends_requests,
             ) = get_trends_data(stats_data, request)
 
-            if features.has(
-                "organizations:performance-trends-issues", organization, actor=request.user
-            ):
-                try:
-                    parse_and_send_regressions_to_plaform(trending_events)
-                except Exception as error:
-                    sentry_sdk.capture_exception(error)
```

Comment:
Maintaining this was more trouble than it was worth especially now that we're no longer relying on this to generate issues.